VersionTLS constants and VersionName(version uint16) are missing

## Problem
Let's say I plan to use a go module like [https://github.com/redis/go-redis](https://github.com/redis/go-redis) in my go project and compile it with TinyGo to a specific target (I'm targeting wasip2 but I guess it can be any target). Also, let's assume that I'm not planning to use any TLS related features in `go-redis`.

Currently, I cannot even compile when I'm using `go-redis` because `VersionTLS` constants and `VersionName(version uint16)` are missing, even though I'm not planning to use them (I grabbed the following code from `redis-go` and added `netDevImplementation`):

```go
package main

import (
	"context"
	"fmt"

	"github.com/redis/go-redis/v9"
	netDevImplementation "path/to/netdev/implementation"
	"tinygo.org/x/drivers/netdev"
)

var ctx = context.Background()

func main() {
	netdev.UseNetdev(netDevImplementation.NewNetdev())
	rdb := redis.NewClient(&redis.Options{
		Addr:     "localhost:6379",
		Password: "", // no password set
		DB:       0,  // use default DB
	})

	err := rdb.Set(ctx, "key", "value", 0).Err()
	if err != nil {
		panic(err)
	}

	val, err := rdb.Get(ctx, "key").Result()
	if err != nil {
		panic(err)
	}
	fmt.Println("key", val)

	val2, err := rdb.Get(ctx, "key2").Result()
	if err == redis.Nil {
		fmt.Println("key2 does not exist")
	} else if err != nil {
		panic(err)
	} else {
		fmt.Println("key2", val2)
	}
	// Output: key value
	// key2 does not exist
}
```

```bash
tinygo build -target=wasip2 --wit-package path/to/wit --wit-world cli main.go
```
```bash
# github.com/redis/go-redis/v9
../../../../go/pkg/mod/github.com/redis/go-redis/v9@v9.7.0/options.go:325:20: undefined: tls.VersionTLS12
```

## Temporarily solution?
I know TLS is yet to be implemented for TinyGo, but I was wondering if we can add `VersionTLS` constants and `VersionName(version uint16)` for now so we can have modules like `go-redis` at least compiling.

### Code to add (`src/crypto/tls/common.go`)
```go
const (
	VersionTLS10 = 0x0301
	VersionTLS11 = 0x0302
	VersionTLS12 = 0x0303
	VersionTLS13 = 0x0304

	// Deprecated: SSLv3 is cryptographically broken, and is no longer
	// supported by this package. See golang.org/issue/32716.
	VersionSSL30 = 0x0300
)

// VersionName returns the name for the provided TLS version number
// (e.g. "TLS 1.3"), or a fallback representation of the value if the
// version is not implemented by this package.
func VersionName(version uint16) string {
	switch version {
	case VersionSSL30:
		return "SSLv3"
	case VersionTLS10:
		return "TLS 1.0"
	case VersionTLS11:
		return "TLS 1.1"
	case VersionTLS12:
		return "TLS 1.2"
	case VersionTLS13:
		return "TLS 1.3"
	default:
		return fmt.Sprintf("0x%04X", version)
	}
}
```
```bash
tinygo build -target=wasip2 --wit-package path/to/wit --wit-world cli main.go
```
Now it compiles and I can run it with a wasm runtime like Wasmtime.
```bash
wasmtime run -Shttp -Sinherit-network -Sallow-ip-name-lookup main.wasm
```
Output as expected:
```bash
key value
key2 does not exist
```

--- a/src/crypto/tls/common.go
+++ b/src/crypto/tls/common.go
@@ -17,6 +17,37 @@ import (
 	"time"
 )
 
+const (
+	VersionTLS10 = 0x0301
+	VersionTLS11 = 0x0302
+	VersionTLS12 = 0x0303
+	VersionTLS13 = 0x0304
+
+	// Deprecated: SSLv3 is cryptographically broken, and is no longer
+	// supported by this package. See golang.org/issue/32716.
+	VersionSSL30 = 0x0300
+)
+
+// VersionName returns the name for the provided TLS version number
+// (e.g. "TLS 1.3"), or a fallback representation of the value if the
+// version is not implemented by this package.
+func VersionName(version uint16) string {
+	switch version {
+	case VersionSSL30:
+		return "SSLv3"
+	case VersionTLS10:
+		return "TLS 1.0"
+	case VersionTLS11:
+		return "TLS 1.1"
+	case VersionTLS12:
+		return "TLS 1.2"
+	case VersionTLS13:
+		return "TLS 1.3"
+	default:
+		return fmt.Sprintf("0x%04X", version)
+	}
+}
+
 // CurveID is the type of a TLS identifier for an elliptic curve. See
 // https://www.iana.org/assignments/tls-parameters/tls-parameters.xml#tls-parameters-8.
 //

